cbor/shared/src/test/scala/fs2/data/cbor/RoundtripTest.scala

```diff
@@ -0,0 +1,26 @@
+package fs2
+package data
+package cbor
+
+import cats.Show
+import fs2.data.cbor.high.CborValue
+import org.scalacheck.Arbitrary
+import weaver.SimpleIOSuite
+import weaver.scalacheck.Checkers
+
+object RoundtripTest extends SimpleIOSuite with Checkers {
```

Comment:
We should extend this kind of test to all types of CBOR values, I did this in http4s-fs2-data and found two bugs already. Left for a follow-up.